examples/snowplow_app.py

```diff
@@ -0,0 +1,58 @@
+import sys
+from snowplow_tracker import (
+    Snowplow,
+    EmitterConfiguration,
+    Subject,
+    TrackerConfiguration,
+    SelfDescribingJson,
+)
+
+
+def get_url_from_args():
+    if len(sys.argv) != 2:
+        raise ValueError("Collector Endpoint is required")
+    return sys.argv[1]
+
+
+def main():
+
+    collector_url = get_url_from_args()
+    # Configure Emitter
+    emitter_config = EmitterConfiguration(buffer_size=5)
+
+    # Configure Tracker
+    tracker_config = TrackerConfiguration(encode_base64=True)
+
+    # Initialise subject
+    subject = Subject()
+    subject.set_user_id("uid")
+
+    Snowplow.create_tracker(
+        namespace="ns",
+        endpoint=collector_url,
+        app_id="app1",
+        subject=subject,
+        tracker_config=tracker_config,
+        emitter_config=emitter_config,
+    )
+
+    Snowplow.get_tracker("ns").track_page_view("https://www.snowplow.io", "Homepage")
```

Comment:
Nit: wouldn't it be better to store the `tracker` object that is returned from `Snowplow.create_tracker` and use that instead of calling `Snowplow.get_tracker` every time?